presto-main/src/main/java/com/facebook/presto/sql/planner/SubqueryPlanner.java

```diff
@@ -361,6 +375,18 @@ private static boolean isAggregationWithEmptyGroupBy(PlanNode planNode)
                 .orElse(false);
     }
 
+    private SubqueryExpression uncoercedSubquery(SubqueryExpression subquery)
```

Comment:
That's what I suspected. We should document that, because it's not at all obvious from just looking at the method.